services/distribution/src/main/java/app/coronawarn/server/services/distribution/dgc/client/JsonSchemaMappingLookup.java

```diff
@@ -0,0 +1,61 @@
+package app.coronawarn.server.services.distribution.dgc.client;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.NoSuchElementException;
+import java.util.Optional;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class JsonSchemaMappingLookup {
+
+  public static final String JSON_SCHEMA_PATH = "dgc";
+  public static final String CCL_JSON_SCHEMA = JSON_SCHEMA_PATH + "/ccl-configuration.json";
+  public static final String DCC_VALIDATION_RULE_JSON_CLASSPATH = JSON_SCHEMA_PATH + "/dcc-validation-rule.json";
+  public static final String ALLOW_LIST_JSON_CLASSPATH =
+      JSON_SCHEMA_PATH + "/dcc-validation-service-allowlist-rule.json";
+
+  static final Map<String, String> businessObjectToJsonSchema = new HashMap<>();
+
+  private static final Logger logger = LoggerFactory.getLogger(JsonSchemaMappingLookup.class);
+
+  /**
+   * Constructor to initialize the schema mappings.
+   */
+  public JsonSchemaMappingLookup() {
+    //initialize all mappings from business objects to the respective schema
+    //TODO: use the fully qualified names instead of objects as map keys,
+    //since we can also have lists of items etc, which makes things really complicated to construct
+    //TODO: switch from types to the requesst URL, since that is uniquely specific to a schema
+    businessObjectToJsonSchema.put("/rules", DCC_VALIDATION_RULE_JSON_CLASSPATH);
+    businessObjectToJsonSchema.put("/bnrules", DCC_VALIDATION_RULE_JSON_CLASSPATH);
+    businessObjectToJsonSchema.put("/cclrules", CCL_JSON_SCHEMA);
+    //businessObjectToJsonSchema.put(/allowlist, DCC_VALIDATION_RULE_JSON_CLASSPATH);
```

Comment:
is there still any **TODO** or is only a bit cleanup required?

Reply:
There was some more cleanup, but now it should be ok.